tap_facebook/__init__.py

```diff
@@ -153,6 +157,8 @@ def should_retry_api_error(exception):
             return True
         elif isinstance(exception, TypeError) and str(exception) == "string indices must be integers":
             return True
+        elif isinstance(exception, Timeout):
```

Comment:
Can we add this in or of one of the above conditions?

Reply:
Updates as per suggestion.